tools/cldr-apps/js/src/views/Waiting.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div id="home">
+        <h1 class='hang'>
+            Waiting for the SurveyTool to startup…
+        </h1>
+        <!-- todo: spinner -->
+        <p>
+            Tried: {{ statusData.triedToStartUp }}
+        </p>
+    </div>
+</template>
+
+<script>
+export default {
+    data: function() {
+        return {
+            statusData: {
+            },
+            attemptedLoadCount: 0,
+            attemptedLoadErr: null
+        }
+    },
+    created: function() {
+        this.fetchStatus();
+    },
+    methods: {
+        fetchStatus: function() {
+            fetch("SurveyAjax?what=status")
+            .then(r => r.json())
+            .then(data => {
+                this.statusData = data;
+                if(!data.triedToStartUp) {
+                    // attempt to fetch /survey
+                    console.log("Attempting fetch of /cldr-apps/survey");
+                    fetch("survey")
```

Comment:
this works to cause ST to startup. Just need the waiting page to retry the status ajax, and call `reload()` when needed